TAF/testScenarios/integrationTest/UC_system_events/metadata_system_events_redis.robot

```diff
@@ -0,0 +1,88 @@
+    And Set Test Variable  ${device_name}  add-system-events
+    And Run Redis Subscriber Progress And Output  ${add_device_topic}  system-events  3
+    When Create 3 Devices For device-virtual
+    Then Multiple 3 System Events Should Match With Source core-metadata, Type device and Action add
+    [Teardown]  Run Keywords  Delete multiple devices by names  @{device_list}
+                ...      AND  Terminate All Processes  kill=True
+
+SystemEventsRedis003-Update Device and Receive Correct System Events
+    Given Set Test Variable  ${update_device_topic}  edgex.system-events.core-metadata.device.update.device-virtual.Virtual-Sample-Profile
+    And Set Test Variable  ${device_name}  update-system-event
+    And Run Redis Subscriber Progress And Output  ${update_device_topic}  system-events
+    And Create Device For device-virtual With Name ${device_name}
+    When Update Device ${device_name} With operatingState=DOWN
+    Then Single System Event Should Match With Source core-metadata, Type device and Action update
+    [Teardown]  Run Keywords  Delete device by name ${device_name}
+                ...      AND  Terminate All Processes  kill=True
+
+SystemEventsRedis004-Update Multiple Devices and Receive Correct System Events
+    Given Set Test Variable  ${update_devices_topic}  edgex.system-events.core-metadata.device.update.device-virtual.Virtual-Sample-Profile
+    And Run Redis Subscriber Progress And Output  ${update_devices_topic}  system-events  3
+    And Create 3 Devices For device-virtual
+    When Update Multiple Devices
+    Then Multiple 3 System Events Should Match With Source core-metadata, Type device and Action update
+    [Teardown]   Run Keywords  Delete multiple devices by names  @{device_list}
+                ...      AND  Terminate All Processes  kill=True
+
+SystemEventsRedis005-Delete Device and Receive Correct System Events
+    Given Set Test Variable  ${delete_device_topic}  edgex.system-events.core-metadata.device.delete.device-virtual.Virtual-Sample-Profile
+    And Set Test Variable  ${device_name}  delete-system-event
+    And Run Redis Subscriber Progress And Output  ${delete_device_topic}  system-events
+    And Create Device For device-virtual With Name ${device_name}
+    When Delete device by name ${device_name}
+    Then Single System Event Should Match With Source core-metadata, Type device and Action delete
+    [Teardown]   Terminate All Processes  kill=True
+
+*** Keywords ***
+Update Multiple Devices
+    Update Device Command-Device-0 With operatingState=DOWN
+    Update Device Command-Device-1 With adminState=LOCKED
+    Update Device Command-Device-2 With label=test-system-event
+
+Single System Event Should Match With Source ${source}, Type ${type} and Action ${action}
+    ${received_content}  grep file  ${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}  Payload
+    Should Not Be Empty  ${received_content}
```

Comment:
```suggestion
    Wait Until Keyword Succeeds  5x  1s  File Should Not Be Empty  ${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}
    ${received_content}  grep file  ${WORK_DIR}/TAF/testArtifacts/logs/${subscriber_file}  Payload
```
Also use on multiple keyword.

Reply:
Thanks for the review comment. Updated.
I will open aother PR on mqtt bus tests.

Reply:
updated